bower.json

```diff
@@ -1,8 +1,9 @@
 {
   "name": "event-infinite-scroll",
-  "version": "0.0.3",
+  "version": "0.1.0",
   "authors": [
-    "chadliu23 <chadliu23@gmail.com>"
+    "chadliu23 <chadliu23@gmail.com>",
+    "Andrea Sonny <andreasonny83@gmail.com>"
```

Comment:
not saying you do or do not deserve to be here, but what constitutes being included in the authors' list? Do you have to be one of the original designers/coders? Can you tweak existing functionality and submit a pull request? Is it entirely up to the repo owner?